src/nunit.analyzers.tests/TestCaseUsage/TestCaseUsageAnalyzerTests.cs

```diff
@@ -100,6 +110,30 @@ public sealed class AnalyzeWhenArgumentIsCorrect
             AnalyzerAssert.Valid<TestCaseUsageAnalyzer>(testCode);
         }
 
+        [TestCaseSource(nameof(SpecialConversions))]
+        public void AnalyzeWhenArgumentIsSpecialConversion(string value, Type targetType)
+        {
+            var testCode = TestUtility.WrapClassInNamespaceAndAddUsing(@"
+    public sealed class AnalyzeWhenArgumentIsSpecialConversion
+    {
+        [TestCase(""" + value + @""")]
+        public void Test(" + targetType.Name + @" a) { }
+    }");
+            AnalyzerAssert.Valid<TestCaseUsageAnalyzer>(testCode);
+        }
+        
+        [Test]
+        public void AnalyzeWhenArgumentIsVerySpecialConversion()
```

Comment:
This test looks the same as the `TestCaseData("2019-10-10", typeof(DateTimeOffset))` case above?

Reply:
Oh you're right, thanks for spotting this! I'd been rushing to get this PR up before my laptop battery died, and I must've missed cleaning up this earlier attempt at a test.